test/cmdlineTests/standard_model_checker_divModSlacks_default_all/output.json

```diff
@@ -1,76 +1,4 @@
 {
-    "auxiliaryInputRequested":
-    {
-        "smtlib2queries":
-        {
-            "0x72b32f47722af7f889ab5740b229c1799f27d0a693995a581c00f65438255a15": "(set-option :produce-models true)
```

Comment:
These auxiliary inputs disappeared because smtlib is now not enabled by default I guess?

Reply:
Exactly.